tests/integration/test_api_version.py

```diff
@@ -31,7 +31,9 @@ def skipif_version(self, nb_client):
     def tag(self, nb_client):
         """Create a tag."""
         tag_name = "Test Tag"
-        tag = nb_client.extras.tags.create(name=tag_name, slug=tag_name.lower().replace(" ", "_"))
+        tag = nb_client.extras.tags.create(
+            name=tag_name, slug=tag_name.lower().replace(" ", "_"), content_types=["dcim.location"]
```

Comment:
```suggestion
            name=tag_name, content_types=["dcim.location"]
```

Reply:
`pynautobot.core.query.RequestError: The request failed with code 400 Bad Request: {'slug': ['This field is required.']`

Reply:
Hmm, may need to wait until 2.0.0b2 is released, this was a fairly recent removal.